distributive/distributive-docker/distributive-docker.json

```diff
@@ -21,6 +21,11 @@
             "Name": "docker systemd service enabled",
             "ID": "systemctlLoaded",
             "Parameters": ["docker.service"]
-        } 
+        },
+        {
+            "Name": "docker disk usage is below 90%",
```

Comment:
Depends on your perspective :) I can change it if you think that's more natural!